README.md

```diff
@@ -1,39 +1,52 @@
+# Errbot - Discord
+
+This is a [Discord](http://discordapp.com) back-end for [Errbot](http://errbot.io).  It allows you to use errbot from Discord to execute commands.
 [![Updates](https://pyup.io/repos/github/gbin/err-backend-discord/shield.svg)](https://pyup.io/repos/github/gbin/err-backend-discord/)
 
-This is a backend for Discord (http://discordapp.com) for errbot (http://errbot.io).
+**This project is currently under development!**
+Active progress is being made under the `development` branch and will be merged to `master` once it is stable.  You are welcome to try the development branch to help test and report any issues encountered with it.
 
-It allows you to use errbot from Discord but it is a work in progress.
+## Installation
+An errbot instance is required to install the discord back-end.  See the errbot installation [documentation](http://errbot.io/en/latest/user_guide/setup.html#option-2-installing-errbot-in-a-virtualenv-preferred) for details.
 
-**This project is in need of a regular user of Discord familiar with Python to take it under his/her wing!**
+### Requirements
+ * Python 3.6 or later
+ * Discord.py 1.2.3 or later
 
-It is Python 3.4+ only.
+### Virtual Environment
+The steps below are to install the discord back-end in errbot's virtual environment.  In the examples below, the virtual environment was set to `/opt/errbot/virtualenv` and errbot initialised in `/opt/errbot`.  The "extra" back-end directory is set to `/opt/errbot/backend`.
 
-## Installation
 
+1. If not already set, set errbot's `BOT_EXTRA_BACKEND_DIR` variable in `/opt/errbot/config.py` to the directory you will use to place additional back-ends.
```

Comment:
not sure `/opt` is the best place to put this

Reply:
`/opt` is only there are an example.  Each environment will have it's own notion of the best place to place the configuration file and it's left to the reader to make the right decision for their setup.